scripts/test/lint.py

```diff
@@ -105,7 +105,7 @@ def check(self, path):
 
         non_required_fields = list()
         for c in metadata.childNodes:
-            if c.nodeName == "#text":
+            if c.nodeName == "#text" or c.nodeName == "#comment":
```

Comment:
[nitpick] What about adding a comment above this line to explain what is does to ensure it is clear to others and it does not get removed by accident in the future? :thinking: Something like:

```suggestion
            # Skip text nodes (whitespace) and comments.
            if c.nodeName == "#text" or c.nodeName == "#comment":
```